### ISSUE TITLE
sk89q-repo in common/pom.xml is not responsive and causing very slow build

### ISSUE THREAD

**Issue by john-jinghai-ma** [Reading-eScience-Centre/edal-java#162]
Is it save to remove that repository from common/pom.xml ? Maven build appears fine without that.

**Comment by guygriffiths** [comment]
Yes, I believe that should be fine.  I think it hosted the `org.khelekore.prtree` code, which was removed from dependencies a long time ago (https://github.com/Reading-eScience-Centre/edal-java/commit/3987c7ddfe5c9a8c09d5abb37e7db091f8fa835c) and brought into the actual codebase, so that some necessary interfaces could be implemented.
